### PR TITLE
Issue #222085 Fix: Vendor Fees page has not proper UI.

### PR DESCRIPTION
**Pull request by Sachintechjoomla**

### CODE REVIEW

File: src/com_tjvendors/admin/views/vendorfee/tmpl/edit_bs5.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @package     TJVendors
+ * @subpackage  com_tjvendors
+ *
+ * @author      Techjoomla <extensions@techjoomla.com>
+ * @copyright   Copyright (C) 2009 - 2019 Techjoomla. All rights reserved.
```
Comment (vaibhavsTekdi) [inline src/com_tjvendors/admin/views/vendorfee/tmpl/edit_bs5.php]: @Sachintechjoomla Correct the year.

File: src/com_tjvendors/admin/views/vendorfee/tmpl/edit_bs2.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+defined('_JEXEC') or die();
+
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Router\Route;
+
+HTMLHelper::addIncludePath(JPATH_COMPONENT . '/helpers/html');
+HTMLHelper::_('bootstrap.tooltip');
+HTMLHelper::_('behavior.formvalidator');
+HTMLHelper::_('formbehavior.chosen', 'select');
```
Comment (vaibhavsTekdi) [inline src/com_tjvendors/admin/views/vendorfee/tmpl/edit_bs2.php:19]: @Sachintechjoomla Is chosen needed?

File: src/com_tjvendors/admin/views/vendorfee/tmpl/edit_bs2.php [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @package     TJVendors
+ * @subpackage  com_tjvendors
+ *
+ * @author      Techjoomla <extensions@techjoomla.com>
+ * @copyright   Copyright (C) 2009 - 2019 Techjoomla. All rights reserved.
```
Comment (vaibhavsTekdi) [inline src/com_tjvendors/admin/views/vendorfee/tmpl/edit_bs2.php]: @Sachintechjoomla correct the year.

File: src/com_tjvendors/admin/views/vendorfee/tmpl/edit_bs2.php (hunk 2 above)
Comment (Sachintechjoomla) [reply src/com_tjvendors/admin/views/vendorfee/tmpl/edit_bs2.php:19]: Yes we need this in joomla 3 for display chosen than select

Comment (Sachintechjoomla) [reply src/com_tjvendors/admin/views/vendorfee/tmpl/edit_bs2.php:19]: > Yes we need this in joomla 3 for display chosen than select

**Pull request merged**